test/alternator/conftest.py

```diff
@@ -13,7 +13,7 @@
 import boto3
 import requests
 import re
-from util import create_test_table, is_aws
+from .util import create_test_table, is_aws
```

Comment:
Previously this was an absolute import. The root (for this "project"), containing all Python modules and packages, was the `test/alternator/` directory. There was a module `util` directly under the root.

Now it is a relative import. `util` is no longer inside the root; the `test` package is. The absolute version of this import would be `from test.alternator.util import ...`. A short version is `from .util` because both `conftest` and `util` are submodules of `test.alternator`.

Note that the relative import version should work in both worlds.